ERROR - 2024-06-14 00:41:53 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'scan_barcode', 3307, '', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 00:41:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: checklogin]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 01:06:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:06:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:06:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:06:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:06:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:07:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:09:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:10:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:10:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:10:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:10:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:10:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:11:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:11:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:11:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:13:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:13:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:13:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:15:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:15:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:16:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:17:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:18:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:18:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:18:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:20:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:25:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:25:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:32:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:32:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:32:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:32:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:32:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:32:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:35:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:35:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:36:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:36:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:36:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:36:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:36:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:37:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:38:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:58:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 01:58:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 01:58:18 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "Packing/halaman_utama.php"
[Method: GET, Route: packing]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Packing/halaman_utama.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Packing/halaman_utama', [], true)
 3 APPPATH\Controllers\PackingController.php(12): view('Packing/halaman_utama')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PackingController->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PackingController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 01:58:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:01:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:01:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:01:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:01:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:05:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-14 02:05:20 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `ma...', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ma...')
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ma...')
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\xampp\htdocs\ScanBarcode\app\Models\AksesorisModel.php(100): CodeIgniter\Model->insert(Array)
#8 D:\xampp\htdocs\ScanBarcode\app\Controllers\AksesorisController.php(93): App\Models\AksesorisModel->insertPDK(Array)
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AksesorisController->inputPDK()
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2024-06-14 02:05:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
[Method: POST, Route: aksesoris/prosesInputPDK]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `master_po` (`id_po`) VALUES (:id_po:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\AksesorisModel.php(100): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\AksesorisController.php(93): App\Models\AksesorisModel->insertPDK([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->inputPDK()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 02:05:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `master_po` (`id_po`) VALUES (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `master_po` (`id_po`) VALUES (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `master_po` (`id_po`) VALUES (:id_po:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\AksesorisModel.php(100): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\AksesorisController.php(93): App\Models\AksesorisModel->insertPDK([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->inputPDK()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 02:05:21 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `master_po` (`id_po`) VALUES (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `master_po` (`id_po`) VALUES (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `master_po` (`id_po`) VALUES (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `master_po` (`id_po`) VALUES (:id_po:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\AksesorisModel.php(100): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\AksesorisController.php(93): App\Models\AksesorisModel->insertPDK([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->inputPDK()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 02:09:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:09:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:10:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:22:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:30:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:30:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:30:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 02:32:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
<<<<<<< HEAD
ERROR - 2024-06-14 03:12:18 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'scan_barcode', 3307, '', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 03:12:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: checklogin]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
ERROR - 2024-06-14 03:13:27 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'scan_barcode', 3307, '', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 03:13:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: checklogin]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
ERROR - 2024-06-14 03:30:29 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'scan_barcode', 3307, '', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 03:30:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: checklogin]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\AuthController.php(22): App\Models\UserModel->login('acc', 'acc')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
ERROR - 2024-06-14 03:33:14 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'scan_barcode', 3307, '', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('aks', 'aks')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('aks', 'aks')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 03:33:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: checklogin]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\AuthController.php(22): App\Models\UserModel->login('aks', 'aks')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
ERROR - 2024-06-14 03:34:01 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'scan_barcode', 3307, '', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1481): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1500): CodeIgniter\Database\BaseConnection->listTables()
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 D:\xampp\htdocs\ScanBarcode\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1481): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1500): CodeIgniter\Database\BaseConnection->listTables()
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 D:\xampp\htdocs\ScanBarcode\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2024-06-14 03:39:46 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'scan_barcode', 3307, '', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acca', 'aaa')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AuthController.php(22): App\Models\UserModel->login('acca', 'aaa')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 03:39:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: checklogin]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\UserModel.php(49): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\AuthController.php(22): App\Models\UserModel->login('acca', 'aaa')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 03:46:47 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "http://localhost:8080/.php"
[Method: POST, Route: checklogin]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('http://localhost:8080/.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('http://localhost:8080/', [], true)
 3 APPPATH\Models\UserModel.php(52): view('http://localhost:8080/')
 4 APPPATH\Controllers\AuthController.php(22): App\Models\UserModel->login('a', 'a')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 03:47:13 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "http://localhost:8080/.php"
[Method: POST, Route: checklogin]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('http://localhost:8080/.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('http://localhost:8080/', [], true)
 3 APPPATH\Models\UserModel.php(52): view('http://localhost:8080/')
 4 APPPATH\Controllers\AuthController.php(22): App\Models\UserModel->login('a', 'a')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 03:48:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:49:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:53:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:57:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:57:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:57:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:57:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:58:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 03:58:20 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "aksesoris/.php"
[Method: POST, Route: aksesoris/prosesInputPO]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('aksesoris/.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('aksesoris/', [], true)
 3 APPPATH\Controllers\AksesorisController.php(54): view('aksesoris/')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->inputPO()
=======
INFO - 2024-06-14 03:21:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:21:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:21:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:38:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 03:39:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 03:39:05 --> BadMethodCallException: Call to undefined method App\Models\MasterPDKModel::getDetailPDK
[Method: GET, Route: aksesoris/dataPDK/2]
in SYSTEMPATH\Model.php on line 932.
 1 APPPATH\Controllers\AksesorisController.php(153): CodeIgniter\Model->__call('getDetailPDK', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPDK('2')
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 03:39:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 03:39:12 --> BadMethodCallException: Call to undefined method App\Models\MasterPDKModel::getDetailPDK
[Method: GET, Route: aksesoris/dataPDK/1]
in SYSTEMPATH\Model.php on line 932.
 1 APPPATH\Controllers\AksesorisController.php(153): CodeIgniter\Model->__call('getDetailPDK', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPDK('1')
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:02:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:02:12 --> BadMethodCallException: Call to undefined method App\Models\MasterPOModel::getPDK
[Method: GET, Route: aksesoris/dataPDK/1]
in SYSTEMPATH\Model.php on line 932.
 1 APPPATH\Controllers\AksesorisController.php(155): CodeIgniter\Model->__call('getPDK', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPDK('1')
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:02:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:04:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:04:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:04:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:05:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:05:58 --> ParseError: Unmatched ')'
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Views\aksesoris\detailPO.php on line 82.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Aksesoris/detailPO', [], true)
 3 APPPATH\Controllers\AksesorisController.php(75): view('Aksesoris/detailPO', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
>>>>>>> 098f757645064b07c60fb47374581f3d173f479f
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
<<<<<<< HEAD
INFO - 2024-06-14 03:59:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:01:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:02:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:02:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:02:19 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "aksesoris.php"
[Method: POST, Route: aksesoris/prosesInputPO]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('aksesoris.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('aksesoris', [], true)
 3 APPPATH\Controllers\AksesorisController.php(55): view('aksesoris')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->inputPO()
=======
INFO - 2024-06-14 04:06:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:06:02 --> ParseError: Unmatched ')'
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Views\aksesoris\detailPO.php on line 82.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Aksesoris/detailPO', [], true)
 3 APPPATH\Controllers\AksesorisController.php(75): view('Aksesoris/detailPO', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
>>>>>>> 098f757645064b07c60fb47374581f3d173f479f
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
<<<<<<< HEAD
INFO - 2024-06-14 04:03:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:03:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:03:40 --> ErrorException: Undefined variable $role
[Method: POST, Route: aksesoris/prosesInputPO]
in APPPATH\Views\Aksesoris\halamanUtama.php on line 1.
 1 APPPATH\Views\Aksesoris\halamanUtama.php(1): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $role', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Views\\Aksesoris\\halamanUtama.php', 1)
 2 SYSTEMPATH\View\View.php(234): include('D:\\xampp\\htdocs\\ScanBarcode\\app\\Views\\Aksesoris\\halamanUtama.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('aksesoris/halamanUtama', [], true)
 5 APPPATH\Controllers\AksesorisController.php(55): view('aksesoris/halamanUtama')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->inputPO()
=======
INFO - 2024-06-14 04:08:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:08:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:08:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:08:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:12:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:16:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:16:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:21:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:21:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:23:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:23:07 --> ErrorException: Undefined array key "id_pdk"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Views\aksesoris\detailPO.php on line 82.
 1 APPPATH\Views\aksesoris\detailPO.php(82): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_pdk"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Views\\aksesoris\\detailPO.php', 82)
 2 SYSTEMPATH\View\View.php(234): include('D:\\xampp\\htdocs\\ScanBarcode\\app\\Views\\aksesoris\\detailPO.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Aksesoris/detailPO', [], true)
 5 APPPATH\Controllers\AksesorisController.php(75): view('Aksesoris/detailPO', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
>>>>>>> 098f757645064b07c60fb47374581f3d173f479f
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
<<<<<<< HEAD
INFO - 2024-06-14 04:20:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:21:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 04:21:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 05:31:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
=======
INFO - 2024-06-14 04:24:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:24:31 --> ErrorException: Undefined array key "id_pdk"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Views\aksesoris\detailPO.php on line 80.
 1 APPPATH\Views\aksesoris\detailPO.php(80): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_pdk"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Views\\aksesoris\\detailPO.php', 80)
 2 SYSTEMPATH\View\View.php(234): include('D:\\xampp\\htdocs\\ScanBarcode\\app\\Views\\aksesoris\\detailPO.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Aksesoris/detailPO', [], true)
 5 APPPATH\Controllers\AksesorisController.php(75): view('Aksesoris/detailPO', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:25:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-14 04:25:13 --> mysqli_sql_exception: Column 'id_po' in field list is ambiguous in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT `id_po`\n...', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id_po`\n...')
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id_po`\n...')
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `id_po`\n...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', Array)
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 04:25:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_po' in field list is ambiguous
[Method: GET, Route: aksesoris/dataPO/2]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 2 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 04:25:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_po' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 4 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 6 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 04:25:13 --> [Caused by] mysqli_sql_exception: Column 'id_po' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 5 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 7 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:25:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-14 04:25:14 --> mysqli_sql_exception: Column 'id_po' in field list is ambiguous in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT `id_po`\n...', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id_po`\n...')
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id_po`\n...')
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `id_po`\n...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', Array)
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 04:25:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_po' in field list is ambiguous
[Method: GET, Route: aksesoris/dataPO/2]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 2 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 04:25:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_po' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 4 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 6 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 04:25:14 --> [Caused by] mysqli_sql_exception: Column 'id_po' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 5 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 7 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:27:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-14 04:27:16 --> mysqli_sql_exception: Column 'id_po' in where clause is ambiguous in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT `master_...', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `master_...')
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `master_...')
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `master_...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', Array)
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 04:27:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_po' in where clause is ambiguous
[Method: GET, Route: aksesoris/dataPO/2]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 2 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 04:27:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_po' in where clause is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 4 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 6 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 04:27:16 --> [Caused by] mysqli_sql_exception: Column 'id_po' in where clause is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 5 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 7 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:27:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-14 04:27:17 --> mysqli_sql_exception: Column 'id_po' in where clause is ambiguous in D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT `master_...', 0)
#1 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `master_...')
#2 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `master_...')
#3 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `master_...', Array, false)
#4 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\ScanBarcode\app\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', Array)
#6 D:\xampp\htdocs\ScanBarcode\app\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
#7 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
#8 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
#9 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\ScanBarcode\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\xampp\htdocs\ScanBarcode\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-06-14 04:27:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_po' in where clause is ambiguous
[Method: GET, Route: aksesoris/dataPO/2]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 2 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 04:27:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_po' in where clause is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 4 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 6 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
CRITICAL - 2024-06-14 04:27:17 --> [Caused by] mysqli_sql_exception: Column 'id_po' in where clause is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT `master_po`.`id_po`
FROM `master_pdk`
JOIN `master_po` ON `master_pdk`.`id_po` = `master_po`.`id_po`
WHERE `id_po` = :id_po:', [...], false)
 5 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Models\MasterPDKModel.php(70): CodeIgniter\Model->__call('get', [])
 7 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:27:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:27:55 --> ErrorException: Undefined array key "master_po.id_po"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Models\MasterPDKModel.php on line 71.
 1 APPPATH\Models\MasterPDKModel.php(71): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "master_po.id_po"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Models\\MasterPDKModel.php', 71)
 2 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:27:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:27:56 --> ErrorException: Undefined array key "master_po.id_po"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Models\MasterPDKModel.php on line 71.
 1 APPPATH\Models\MasterPDKModel.php(71): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "master_po.id_po"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Models\\MasterPDKModel.php', 71)
 2 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:29:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:29:09 --> ErrorException: Undefined array key "master_po.id_po"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Models\MasterPDKModel.php on line 71.
 1 APPPATH\Models\MasterPDKModel.php(71): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "master_po.id_po"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Models\\MasterPDKModel.php', 71)
 2 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:29:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:29:09 --> ErrorException: Undefined array key "master_po.id_po"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Models\MasterPDKModel.php on line 71.
 1 APPPATH\Models\MasterPDKModel.php(71): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "master_po.id_po"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Models\\MasterPDKModel.php', 71)
 2 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:29:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:29:10 --> ErrorException: Undefined array key "master_po.id_po"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Models\MasterPDKModel.php on line 71.
 1 APPPATH\Models\MasterPDKModel.php(71): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "master_po.id_po"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Models\\MasterPDKModel.php', 71)
 2 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:29:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:29:10 --> ErrorException: Undefined array key "master_po.id_po"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Models\MasterPDKModel.php on line 71.
 1 APPPATH\Models\MasterPDKModel.php(71): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "master_po.id_po"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Models\\MasterPDKModel.php', 71)
 2 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:29:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-14 04:29:10 --> ErrorException: Undefined array key "master_po.id_po"
[Method: GET, Route: aksesoris/dataPO/2]
in APPPATH\Models\MasterPDKModel.php on line 71.
 1 APPPATH\Models\MasterPDKModel.php(71): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "master_po.id_po"', 'D:\\xampp\\htdocs\\ScanBarcode\\app\\Models\\MasterPDKModel.php', 71)
 2 APPPATH\Controllers\AksesorisController.php(65): App\Models\MasterPDKModel->getIdPO('2')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AksesorisController->detailPO('2')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AksesorisController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\xampp\\htdocs\\ScanBarcode\\public\\index.php')
INFO - 2024-06-14 04:29:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 05:54:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 05:54:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 05:56:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 05:56:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
>>>>>>> 098f757645064b07c60fb47374581f3d173f479f
INFO - 2024-06-14 07:18:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 07:18:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 07:18:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 07:18:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 07:18:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-14 07:58:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
